options/options_test.go

```diff
@@ -128,6 +128,17 @@ func TestNew(t *testing.T) {
 				}
 			}
 
+			if tt.repo != "" {
+				os.Setenv(EnvInputResultsFormat, tt.resultsFormat)
+			}
+
+			_, resultsfileEnvExists := os.LookupEnv(EnvInputResultsFile)
```

Comment:
Any reason to assert whether this is set or not before setting this env var?

Reply:
Removed, this was a remnant from before